sqlx-postgres/src/type_checking.rs

```diff
@@ -24,6 +24,8 @@ impl_type_checking!(
 
         sqlx::postgres::types::PgInterval,
 
+        Vec<sqlx::postgres::types::PgInterval> | &[sqlx::postgres::types::PgInterval],
```

Comment:
Please move this down here with the rest of the array types: https://github.com/launchbadge/sqlx/blob/76b627b0a71f7e590a1146dfe71bf7f487c92b9d/sqlx-postgres/src/type_checking.rs#L101